### PR TITLE
user search ordering fixes

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

File: go/service/usersearch.go [hunk 1]
```diff
@@ -327,13 +327,14 @@ func (h *UserSearchHandler) UserSearch(ctx context.Context, arg keybase1.UserSea
 		if err != nil {
 			mctx.Warning("Failed to do contacts search: %s", err)
 		} else {
-			res = append(res, contactsRes...)
 			// Sort first - we are going to be deduplicating on usernames,
 			// entries with higher score have precedence.
-			sort.Slice(res, func(i, j int) bool {
```
Comment (zapu) [inline go/service/usersearch.go]: sorting everything was intentional, do we want to start displaying contacts at the end of the list even if they are more relevant to the query?

Comment (heronhaye) [reply go/service/usersearch.go]: hm. ok, how about we rewrite the scores of all the followees to be at least what the previous one was? then sort as normal

Comment (heronhaye) [reply go/service/usersearch.go]: /shrug let's do this until we get a better design

Comment (zapu) [reply go/service/usersearch.go]: sounds good, I misunderstood at first. thanks for explaining

Comment (zapu) [reply go/service/usersearch.go]: it would probably be ideal to fix textsearchd but lets not worry about this right now

**Pull request merged**